typescript/packages/plugins/solana/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@goat-sdk/plugin-solana-actions",
```

Comment:
I'd name this `solana-nfts` and keep it just to solana NFTs actions

Reply:
Easier to merge afterwards if we think we would benefit from a larger solana actions package

Reply:
Also the name of the dir should be solana-nfts